umi/sumi/tests/test_regif.py

```diff
@@ -28,22 +25,13 @@ def test_regif(sumi_dut, apply_atomic, random_seed, sb_umi_valid_mode, sb_umi_re
     print("### Starting test ###")
 
     # regif accesses are all 32b wide and aligned
-    for _ in range(n):
-        addr = np.random.randint(0, 512) * 4
+    for _ in range(10):
```

Comment:
why reduce this to 10? from 100?

Reply:
More random transactions added little/zero value in the test.